### ISSUE TITLE
Twitter has new endpoint for counting shares.

### ISSUE THREAD

**Issue by filipdanisko** [olahol/socialcount.js#2]
https://dev.twitter.com/rest/reference/get/search/tweets

Any ideas how to implement this? Old one is deprecated.

**Comment by olahol** [comment]
Thanks for the heads up Filip, I will implement this as soon as possible.

**Comment by olahol** [comment]
Looks like they have removed twitter counts :disappointed: 

https://blog.twitter.com/2015/hard-decisions-for-a-sustainable-platform

I will be removing Twitter from socialcount.js for the time being.

**Comment by filipdanisko** [comment]
Yes you are right, it seems like only option now is to use own backend solution. With gnip api http://support.gnip.com/ or this rest api https://dev.twitter.com/rest/reference/get/search/tweets.

But this solution seems like out of scope of this project.
